pythonBasics3Test.py

```diff
@@ -1,47 +1,53 @@
-
 import unittest
 import pythonBasics3
 
 
 class TestPythonBasicsOne(unittest.TestCase):
 
-#Test case for ends_with_consonant
-    def test_ends_with_consonant(self):
+#Test case for starts_with_non_number
+    def test_starts_with_non_number(self):
 
-        self.assertEqual(pythonBasics3.ends_with_consonant("Once upon a dark, cold night"), True)
+        self.assertEqual(pythonBasics3.starts_with_non_number("Once upon a time, I was 18"), True)
 
-        self.assertEqual(pythonBasics3.ends_with_consonant("What did Steve just say"), True)
+        self.assertEqual(pythonBasics3.starts_with_non_number("5 weekdays each week"), False)
 
-        self.assertEqual(pythonBasics3.ends_with_consonant("I once swam with sharks"), True)
+        self.assertEqual(pythonBasics3.starts_with_non_number("-5 is what I got on my quiz"), True)
```

Comment:
Why is -5 a non number?

Reply:
The goal of this is to be able to use character classes. The first character (`-`) is not part of the digit character class (`\d`). Therefore, the first character is not a number in this case.